crates/rolldown/src/utils/reserved_names.rs

```diff
@@ -0,0 +1,65 @@
+pub const RESERVED_NAMES: [&str; 62] = [
+  "await",
+  "break",
+  "case",
+  "catch",
+  "class",
+  "const",
+  "continue",
+  "debugger",
+  "default",
+  "delete",
+  "do",
+  "else",
+  "enum",
+  "export",
+  "extends",
+  "false",
+  "finally",
+  "for",
+  "function",
+  "if",
+  "implements",
+  "import",
+  "in",
+  "instanceof",
+  "interface",
+  "let",
+  "new",
+  "null",
+  "package",
+  "private",
+  "protected",
+  "public",
+  "return",
+  "super",
+  "switch",
+  "static",
+  "this",
+  "throw",
+  "true",
+  "try",
+  "typeof",
+  "var",
+  "void",
+  "while",
+  "with",
+  "yield",
+  // ECMAScript 6 (ES6) keywords
+  "abstract",
+  "boolean",
+  "byte",
+  "char",
+  "double",
+  "final",
+  "float",
+  "goto",
+  "int",
+  "long",
+  "native",
+  "short",
+  "synchronized",
+  "throws",
+  "transient",
+  "volatile",
```

Comment:
Are these ... java keywords?

Reply:
Yeah. It is es3 RESERVED WORDS.

Reply:
They are not part of the current ecma spec https://tc39.es/ecma262/#prod-ReservedWord

Reply:
> Are these ... java keywords?

```
abstract, boolean, byte, char, double, final, float, goto, int, long, native, short, synchronized, throws, transient, volatile
```
These are reserved words in ES3, but they can be used safely in ES6 and later versions. Do they need to be removed?

Reply:
> They are not part of the current ecma spec https://tc39.es/ecma262/#prod-ReservedWord

Ok, I will remove these keywords.

Reply:
Yeah. esbuild https://github.com/evanw/esbuild/blob/main/internal/js_lexer/js_lexer.go#L162-L212 and rollup https://github.com/rollup/rollup/blob/master/src/utils/RESERVED_NAMES.ts also not include them.

Reply:
@Boshen @underfin keep keywords consistent with rollup